src/Storage/Database/Prefill/RecordContentGenerator.php

```diff
@@ -453,9 +456,21 @@ private function getRandomTags($count = 5)
      *
      * @return array|null
      */
-    private function getRandomImage($type)
+    private function getRandomImage($type, $placeholder = null)
```

Comment:
`$placeholder` doesn't need a default, and does need a `@param string|null $placeholder` in the docblock